dali/daliadmin/daadmin.cpp

```diff
@@ -1103,28 +1103,33 @@ int dfsverify(const char *name,CDateTime *cutoff, IUserDescriptor *user)
 
 //=============================================================================
 
-void setprotect(const char *filename, const char *callerid, IUserDescriptor *user)
+bool setprotect(const char *filename, const char *callerid, IUserDescriptor *user, StringBuffer &out)
```

Comment:
You may keep the 'void' and remove the 'return true;' below -- similar to the 'setdfspartattr'.

Reply:
Changed it back to void and fixed it in other areas.  Works just like setdfspartattr now.